src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -152,6 +159,7 @@ internal sealed override ImmutableArray<string> NotNullWhenFalseMembers
             _property = property;
             _isAutoPropertyAccessor = false;
             _isExpressionBodied = true;
+            _containsFieldKeyword = NodeContainsFieldKeyword(syntax);
```

Comment:
>NodeContainsFieldKeyword(syntax); [](http://example.com/codeflow?start=36&length=33)

Several comments:
- Do we support field keywords in indexers?
- Probably should look specifically in bodies rather than in entire accessor declaration.
- Body vs. ExpressionBody

Fine to add a prototype comment.